src/mappings/index.ts

```diff
@@ -299,7 +300,8 @@ async function list(remark: RemarkResult, { store }: Context) {
 
     logger.success(`[LIST] ${nft.id} from ${remark.caller}`)
     await store.save(nft)
-    await createEvent(nft, RmrkEvent.LIST, remark, String(price), store)
+    const event = nft.price === 0n ? RmrkEvent.UNLIST : RmrkEvent.UNLIST
```

Comment:
shouldn't it be "LIST" on right side?
```suggestion
    const event = nft.price === 0n ? RmrkEvent.UNLIST : RmrkEvent.LIST
```